Makefile

```diff
@@ -0,0 +1,27 @@
+AS := nasm
+ASFLAGS := -fbin
+
+SRC_ASM = $(patsubst %.asm,%.386,$(wildcard src/386/*.asm))
+
+TARGET = EnzOS.386
+DISKIMG = disk.raw
+
+all: $(TARGET)
+	@echo "OS built successfully"
+
+$(TARGET): $(SRC_ASM)
+	cp $^ $@
+
+%.386: %.asm
+	$(AS) $(ASFLAGS) -o "$@" "$^"
+
+image: $(TARGET)
+	rm -f disk.raw
+	dd if=/dev/zero of=$(DISKIMG) bs=1M count=8
+	$(GOPATH)/bin/fdisk mbr -create -bootcode $^ $(DISKIMG)
```

Comment:
Depends on [fdisk of enzo utilities](https://github.com/tiago4orion/enzo/tree/feat/fdisk/cmd/fdisk).